### PR TITLE
Fix migration key

### PR DESCRIPTION
**Pull request by deviantintegral**
This fixes hardcoding the database key, as well as a missing migrate dependency.

### CODE REVIEW

File: migrations/d7_media_theplatform_mpx_source_field_instance.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 id: d7_media_theplatform_mpx_source_field_instance
 label: Source field instance for mpx objects
 migration_group: media_theplatform_mpx
+migration_tags:
+  - 'Drupal 7'
```
Comment (juampynr) [inline migrations/d7_media_theplatform_mpx_source_field_instance.yml:5]: Can you add the Configuration tag? TugBoat runs these first.

File: migrations/d7_media_theplatform_mpx_source_field_storage.yml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 id: d7_media_theplatform_mpx_source_field_storage
 label: Source field for mpx objects
 migration_group: media_theplatform_mpx
+# This migration cannot be attached to the Drupal 7 tag as it uses the Empty
+# plugin, which doesn't define the required source_module key.
+#migration_tags:
+#  - 'Drupal 7'
```
Comment (juampynr) [inline migrations/d7_media_theplatform_mpx_source_field_storage.yml:7]: Is there a ticket for that? Can I help?

Comment (deviantintegral) [reply migrations/d7_media_theplatform_mpx_source_field_storage.yml:7]: I think this is a core issue. Empty literally means "no source" but core assumes all migrations from Drupal 7 have a source. I'm starting to think that perhaps we're overusing this tag in our migrations.

File: migrations/d7_media_theplatform_mpx_source_field_instance.yml (hunk 1 above)
Comment (deviantintegral) [reply migrations/d7_media_theplatform_mpx_source_field_instance.yml:5]: Done!

**Pull request merged**